data/mods/No_Hope/Mapgen/mil_base_z0.json

```diff
@@ -557,7 +557,7 @@
         { "vehicle": "military_vehicles", "x": 79, "y": 42, "chance": 25, "status": 1, "rotation": 90 },
         { "vehicle": "military_vehicles", "x": 86, "y": 13, "chance": 25, "status": 1, "rotation": 270 },
         { "vehicle": "military_vehicles", "x": 79, "y": 54, "chance": 25, "status": 1, "rotation": 270 },
-        { "vehicle": "forklift", "x": 62, "y": 62, "chance": 80, "rotation": 90 }
+        { "vehicle": "forklift", "x": 62, "y": 62, "chance": 80, "status": 0, "rotation": 90 }
```

Comment:
> Comment from Night-Pryanik:
>"Not needed. The default status is -1, which means it's lightly damaged. I see no reason for a forklift to be ideally undamaged with status 0."

@Night-Pryanik 

I went two ways on this. My ultimate thinking was that since the forklift is safely stored in a closed and locked warehouse _where all other items are untouched and undamaged_, it would most likely have been untouched since the cataclysm began and therefore in perfect condition.
If you still disagree, I'll certainly change it to status -1.

Reply:
I just thought that as this forklift was used to move crates and pallets with all this stuff, it should have some wear and tear after all this work.

Reply:
That makes sense. What I'll do is set status to -1 and generate a few tests and see how badly beaten up it is.
As long as it's only a bit dinged up and not nonfunctional, that would make sense to me.

Reply:
Test results: while the forklift is in a lot worse condition than I would prefer, in most cases it's in operable condition, with only 1 in 10 cases where it wasn't due to broken controls. Which is reasonable, since maybe it was broken in the emergency and they didn't have time to fix it.
Changing all vehicle statuses to -1.